python/across_api/base/fov.py

```diff
@@ -0,0 +1,382 @@
+# Copyright © 2023 United States Government as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+# All Rights Reserved.
+
+from typing import Optional, Union
+import astropy_healpix as ah  # type: ignore
+import astropy.units as u  # type: ignore
+from astropy.io.fits import FITS_rec  # type: ignore
+import numpy as np
+from astropy.coordinates import SkyCoord, angular_separation, spherical_to_cartesian  # type: ignore
+from astropy.time import Time  # type: ignore
+import healpy as hp
+from .constraints import EarthLimbConstraint, get_slice
+from .common import ACROSSAPIBase
+from .footprint import Footprint
+from .pointing import PointingBase
+from .ephem import EphemBase
+
+HEALPIX_MAP_EVAL_ORDER = 9
+
+
+# Some math stuff for calculating the probability inside for circular error
+# regions
+def normal_pdf(x, standard_deviation):
+    """
+    Calculate the probability density function (PDF) of a standard normal distribution (mean=0).
+
+    Parameters
+    ----------
+    x
+        The value at which to calculate the PDF.
+    standard_deviation
+        The standard deviation of the normal distribution.
+
+    Returns
+    -------
+    pdf
+        The probability density at the given x.
+    """
+
+    # Calculate the exponent term
+    exponent = -(x**2) / (2 * standard_deviation**2)
+
+    # Calculate the PDF using the simplified formula
+    pdf = (1 / (standard_deviation * np.sqrt(2 * np.pi))) * np.exp(exponent)
+
+    return pdf
+
+
+def healpix_map_from_position_error(
+    skycoord: SkyCoord,
+    error_radius: u.Quantity,
+    nside=hp.order2nside(HEALPIX_MAP_EVAL_ORDER),
+) -> np.ndarray:
+    """
+    For a given sky position and error radius, create a HEALPix map of the
+    probability density distribution.
+
+    Parameters
+    ----------
+    skycoord
+        The sky position for which to create the probability density
+        distribution.
+    error_radius
+        The 1 sigma error radius for the sky position.
+    nside
+        The NSIDE value for the HEALPix map. Default is 512.
+
+    Returns
+    -------
+    prob
+        The probability density distribution HEALPix map.
+    """
+    # Create HEALPix map
+    hp = ah.HEALPix(nside=nside, order="nested")
+
+    # Find RA/Dec for each pixel in HEALPix map
+    hpra, hpdec = hp.healpix_to_lonlat(range(ah.nside_to_npix(nside)))
+
+    # Find angular distance of each HEALPix pixel to the skycoord
+    distance = angular_separation(hpra, hpdec, skycoord.ra, skycoord.dec).to(u.deg)
+
+    # Create the probability density distribution HEALPix map
+    prob = normal_pdf(distance, error_radius).value
```

Comment:
Consider instead `scipy.stats.norm`:

```suggestion
    prob = stats.norm(scale=error_radius).pdf(distance)
```